workspaces/bulk-import/plugins/bulk-import/src/components/Repositories/AddedRepositoryTableRow.tsx

```diff
@@ -71,16 +78,26 @@ export const AddedRepositoryTableRow = ({
   data: AddRepositoryData;
 }) => {
   const classes = useStyles();
-
+  const importFlow = useImportFlow();
   return (
     <TableRow hover>
       <TableCell component="th" scope="row" className={classes.tableCellStyle}>
-        {data.repoName}
+        {importFlow === 'scaffolder' ? (
```

Comment:
We should have also an enum for the importFlow. Not required for now, but would be nice if you can raise a follow-up PR.

Reply:
https://issues.redhat.com/browse/RHDHBUGS-2109